lib/utils/type.js

```diff
@@ -40,6 +40,18 @@ async function detectTypeUsingFileCommand(renditionPath) {
     return result;
 }
 
+/**
+ * Make sure the mimetype we return can be understood by other libraries/software
+ * @param {*} mimetype Input mimetype to normalize
+ * @returns Normalized mimetype
+ */
+function normalizeMimetype(mimetype){
```

Comment:
qs, what if the source mimetype is actually `image/svg`? would aem want it to stay that way or be mapped to `+xml`

Reply:
I think they always need the `+xml` for further processing.